dnf/transaction_sr.py

```diff
@@ -187,14 +187,16 @@ class TransactionReplay(object):
     def __init__(
         self,
         base,
-        fn,
+        fn="",
```

Comment:
Can you please just name the argument `filename` instead of `fn`? It's a bit more important since it's now mutually exclusive with the `data` and they both should be self-descriptive, not abbreviated. Otherwise looks good.

Reply:
Changed. I hope it did not break anything and the tests will pass.